pkg/server/scrub.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/davecgh/go-spew/spew"
```

Comment:
Do we really need to use this?

Reply:
The spec was formatted via `spew` in the original code, so I kept it here for the formatting, since it offered a nice return type.
I can remove it and make the return just a regular `interface{}`

Reply:
In that case, this is fine.